src/main/java/com/hotel/interceptors/ApiKeyAuthInterceptor.java

```diff
@@ -22,39 +21,39 @@
 public class ApiKeyAuthInterceptor implements ServerInterceptor {
 
 
-    public boolean validateSignature(String token)  throws Exception{
+    public boolean validateSignature(String token) throws NoSuchAlgorithmException, InvalidKeySpecException {
         String[] chunks = token.split("\\.");
         SignatureAlgorithm sa = SignatureAlgorithm.RS256;
-        BigInteger mod = new BigInteger("19000812822426683865012163618027310378487395154330304998855217229900382730200283232802318000193794545462996454857896773995909120073180895021730312708474638199242368948617010756177040604721432026065868235078315166342546553887368687889480198247689830862136303023640201198180314337176952541013193315987840899973486485532199280808520596592328947878088072286427483154626278976886368730181756526507275308753625752548958224081969926946453900525575829301891927935306879737770979639402074921073866173771711955654133667856118067012688187894392311708798013285013475533536647677724106088022279193658129435945930408108920563375353");
-        BigInteger publicExp = new BigInteger("65537");
```

Comment:
need to use values form vault for publicExp and mod

Reply:
already resolved